.github/workflows/dev-portal-preview.yml

```diff
@@ -40,10 +39,15 @@ jobs:
       - name: Checkout hashicorp/web-unified-docs
         uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
 
+      - name: Add commit link to summary
```

Comment:
This adds this to the github action summary:

![Screenshot 2025-03-13 at 10 09 10 AM](https://github.com/user-attachments/assets/138d7666-69bf-4210-9e45-068f6dec3e96)

Which links directly to the commit the PR which triggered the build. Which is very useful for debugging. Github doesn't add this bt default because actions can be triggered by so many different things. But we know that this specific action is only ever triggered by PRs.